recipes/libpng/all/test_package/test_package.c

```diff
@@ -3,7 +3,7 @@
 #include "png.h"
 #include "zlib.h"
 
-int main()
+int main(void)
```

Comment:
does it trigger a warning? afaik void could be omitted safely

Reply:
yes, you're correct, I did that instinctively